aws/resource_aws_ram_resource_share_accepter.go

```diff
@@ -233,12 +228,47 @@ func resourceAwsRamResourceShareAccepterDelete(d *schema.ResourceData, meta inte
 	return nil
 }
 
-func resourceAwsRamResourceShareGetInvitation(conn *ram.RAM, resourceShareARN, status string) (*ram.ResourceShareInvitation, error) {
+func resourceAwsRamResourceShareAccepterGetInvitation(conn *ram.RAM, resourceShareARN, status string) (*ram.ResourceShareInvitation, error) {
+	var invitation *ram.ResourceShareInvitation
+
+	// Retry for Ram resource share invitation eventual consistency
+	err := resource.Retry(1*time.Minute, func() *resource.RetryError {
+		i, err := findResourceShareInvitation(conn, resourceShareARN, status)
+		invitation = i
```

Comment:
Not a golang expert, but is there a reason `i` is used here and then immediately assigned to `invitation` instead of just using `invitation, err :=` ?

Reply:
No, actually you cannot substitute the returned output for the outer `invitation` using `:=`...

Reply:
Ah, TIL.  Thanks!